### PR TITLE
8303911: [testbug] Some unit tests call System.exit

### PR DESCRIPTION
**Pull request by lukostyra**
Changes added to this commit resolve cases where `System.exit()` was used to leave tests abruptly. These tests, if System.exit() is hit, could potentially create some unwanted behavior from JTest (ex. silently skipping these tests instead of failing them).

There were some files which I did not modify, mostly because they were a part of a test sub-app, or tests would require major rework to make it work.

Two FXML tests were performance tests already written as separate apps, so I moved them to appropriate directory in tests/performance.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8303911](https://bugs.openjdk.org/browse/JDK-8303911): [testbug] Some unit tests call System.exit

### Reviewers
 * [Andy Goryachev](https://openjdk.org/census#angorya) (@andy-goryachev-oracle - Committer)
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx.git pull/1101/head:pull/1101` \
`$ git checkout pull/1101`

Update a local copy of the PR: \
`$ git checkout pull/1101` \
`$ git pull https://git.openjdk.org/jfx.git pull/1101/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1101`

View PR using the GUI difftool: \
`$ git pr show -t 1101`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/1101.diff">https://git.openjdk.org/jfx/pull/1101.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jfx/pull/1101#issuecomment-1515175773)

### CODE REVIEW

File: tests/system/src/test/java/test/com/sun/javafx/image/impl/ImageRaceTest.java [hunk 1]
```diff
@@ -125,8 +126,7 @@ static void init(String[] args) {
                 } else if (arg.equalsIgnoreCase("-verbose")) {
                     verbose = true;
                 } else {
-                    System.err.println("Unrecognized argument: "+arg);
-                    System.exit(-1);
+                    fail("Unrecognized argument: "+arg);
```
Comment (andy-goryachev-oracle) [inline tests/system/src/test/java/test/com/sun/javafx/image/impl/ImageRaceTest.java:129]: maybe add spaces: "_+_arg

Comment (lukostyra) [reply tests/system/src/test/java/test/com/sun/javafx/image/impl/ImageRaceTest.java:129]: Could be done indeed while this line is edited.

**Comment by lukostyra** [comment]
I also noticed I did not update some copyright boilerplates with current dates

**Pull request closed without merging**